packages/repositories.config

```diff
@@ -1,10 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <repositories>
-  <repository path="..\Octokit\packages.config" />
+  <repository path="../Octokit.Reactive/packages.config" />
```

Comment:
Actually, I was doing this on Xamarin Studio on a OS X. :disappointed: 

But still, why didn't it happened to all the items?